### PR TITLE
Client communication

### PR DESCRIPTION
**Pull request by grondinjc**
I had to change controller to return a text position ...

### CODE REVIEW

**Comment by grondinjc** [comment]
@vphilippon Here, a big one

**Comment by vphilippon** [comment]
I'll take a look tonight

File: src/static/css/style.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.codeInput {
+	background-color: transparent;
```
Comment (vphilippon) [inline src/static/css/style.css:2]: @grondinjc There seems to be a Tab here (or overindent)

File: src/static/css/style.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    border: 0px;
+    outline: none;
+    -webkit-box-shadow: none;
+    -moz-box-shadow: none;
+    box-shadow: none;
+    color:transparent;
+    cursor:default;
+    width: 0px;
+	height: 0px;
```
Comment (vphilippon) [inline src/static/css/style.css:11]: @grondinjc Overindent

File: src/static/css/style.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+.codeInputDebug {
+    width: 100%;
+	height: 300px;
```
Comment (vphilippon) [inline src/static/css/style.css:16]: @grondinjc Overindent

File: src/static/css/style.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+.codeDisplay {
+	border-style: solid;
```
Comment (vphilippon) [inline src/static/css/style.css:20]: @grondinjc Overindent

**Comment by vphilippon** [comment]
:+1: 
Merging

**Pull request merged**